java-client/src/main/java/com/cumulocity/sdk/client/messaging/notifications/TokenClaims.java

```diff
@@ -21,6 +21,10 @@ public class TokenClaims extends BaseResourceRepresentation {
 
     private long exp;
 
+    private boolean shared;
+
+    private boolean nonPersistent;
```

Comment:
Can we just name it `volatile`? Negation is always harder to read.

Reply:
No - volatile is a key word in Java. We had isVolatile but had to jump through so many hops to get the REST marshalling to understand that. I think nonPersistent is closer to Pulsar and better than volatile - which sounds like a free for all.